ui/src/app.tsx

```diff
@@ -4,6 +4,7 @@ import { themeGlobals } from '@/theme/globals';
 import { Home, Mint } from './views';
 import { SVGTestScreen } from './views/svg-test'; // TODO: remove when done
 import { ConnectKitButton } from 'connectkit';
+import { MintTest } from './views/mint-test';

```

Comment:
Maybe we can remove this? cause we already integrated the mint function to the flow

Reply:
Leaving it just as an example about integrations stuff, we must remove it in the future.